scheduler/roles/k8s_habana_container_runtime/tasks/change_containerd_runtime.yml

```diff
@@ -0,0 +1,65 @@
+# Copyright 2024 Intel Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+- name: Uncomment habana-container-runtime config mount_accelerators line
+  lineinfile:
```

Comment:
please update all the modules with ansible.builtin commands 
Ex: instead of lineinfile use ansible.builtin.lineinfile . replicate the same for other tasks in this file

Reply:
Done